libsolidity/formal/SMTEncoder.cpp

```diff
@@ -1158,6 +1161,12 @@ void SMTEncoder::initializeFunctionCallParameters(CallableDeclaration const& _fu
 			}
 }
 
+void SMTEncoder::initializeStateVariables(ContractDefinition const& _contract)
+{
+	for (auto var: _contract.stateVariablesWithInheritance())
```

Comment:
Is it fine that this also returns constants?

Reply:
Yep that's intended.